### ISSUE TITLE
Playing ofline (without internet) not working

### ISSUE THREAD

**Issue by tristanzwart** [AfoninZ/MultiMC5-Cracked#116]
### Operating System

Windows

### Description of bug

The situation is as follows: I start the launcher and download a version (mc 1.17 vanilla) then I started it once. Next, I went somewhere else opened the same instance started it offline but it tried to update a file 1.17.json (asset-index) so it did not download because I have no internet there. As a result of that mc 1.17.1 did not start.

### Steps to reproduce

Create a new instance of vanilla 1.17.1 start it once then go offline and then start it again (play offline button) and it should show the same error.

### Suspected cause

It tries to download a file which it can't because you are offline. Instead, if it can't download it should just continue with a warning of some sort.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by Samisafool** [comment]
Try the "play offline" button?

**Comment by Slddev** [comment]
> Try the "play offline" button?

Read the whole thing, he tried the play offline button,

**Comment by tristanzwart** [comment]
> Try the "play offline" button?

Yes I did as I already said It trys to opdate asset-index even though I had started it previously
